scss/components/content.scss

```diff
@@ -0,0 +1,73 @@
+
+#content {
+  @include grid($content-columns, $offset: grid-offset($nav-columns + 1));
+  @include even-shadow($light-shadow, $shadow-small-size);
+  background: #FFF;
+  padding: $content-padding;
+  margin-top: $content-margin-top;
+
+  h1, h2, p, pre {
+    margin-bottom: $content-h1-margin-bottom;
+  }
+
+  p:last-child {
+    margin-bottom: $content-h1-margin-bottom + 15px;
+  }
+
+  h1 {
+    @include styleguide(xlarge headline);
+    font-size: $headline-font-size;
```

Comment:
Nothing wrong, just preferred the way it looked with that specific size.  I will remove the font-size override.